### DISCUSSION TITLE
Duvida na tela de Cobrança

### DISCUSSION THREAD

**Discussion by valdenirrocha** [RamonSilva20/mapos#2582] [General]
Olá, eu baixei o sistema e estou navegando nele para entender como funciona, não entendi a parte da tela de Cobrança, não tem botão para cadastrar e gerenciar e também não consegui ver a ligação (se é que tem) com outras telas (Os, Vendas e Lançamentos). Gostaria de saber como usar esta tela...

**Comment by maycon1977** [comment]
Você inseriu dados como cliente, produto, serviço, fez O.S., venda, etc?

Precisa cadastrar alguns itens em cada tela para poder simular o uso e
aprender a usar de forma correta...

*Atenciosamente,*
*Maycon Martins Teixeira ***@***.***_rio)*

*Telefone/**WhatsApp**: +55 (
<https://api.whatsapp.com/send?phone=5521988078360&text=Ol%C3%A1%2C%20Maycon.%20Sou%20um%20cliente%20da%20Press%C3%A3oRio.%20Pode%20me%20ajudar%3F>**21)
98807-8360
<https://api.whatsapp.com/send?phone=5521988078360&text=Olá%2C%20Maycon.%20Sou%20um%20cliente%20da%20PressãoRio.%20Pode%20me%20ajudar%3F>*
Facebook: http://www.facebook.com/maycon.rj
*Twitter: https://twitter.com/jabour_rio <https://twitter.com/jabour_rio>*


Em seg., 4 de nov. de 2024 às 11:46, Valdenir ***@***.***>
escreveu:

**Reply by valdenirrocha** [reply]
Sim Mycon, cadastrei clientes, fornecedores, criei varias OS, vendas e lançamentos, quando vou nesta tela de cobrança ela esta vazia e sem nenhum botao...
